ci/playbooks/kuttl/deploy-deps.yaml

```diff
@@ -32,18 +32,28 @@
       loop_control:
         label: "{{ item }}"
 
-    - name: Ensure that the isolated net was configured for crc
-      ansible.builtin.assert:
-        that:
-          - crc_ci_bootstrap_networks_out is defined
-          - "'crc' in crc_ci_bootstrap_networks_out"
-          - "'default' in crc_ci_bootstrap_networks_out['crc']"
+    - name: set facts for further usage within the framework
+      vars:
+        _crc_hostname: "{{ cifmw_crc_hostname | default('crc') }}"
```

Comment:
This is proablly a little more maintainabel since the var wont leak into the outer socope like a fact would.

using a block to scope it to multiple tasks also makes sense.